src/figuresClasses.ts

```diff
@@ -1,11 +1,85 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: string = 'triangle';
+
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    const sides = [a, b, c];
+    const maxSide = Math.max(a, b, c);
+    const leftSides = sides.filter((side) => side !== maxSide);
+    const sumLeftSides = leftSides.reduce((acc, side) => acc + side);
```

Comment:
In the Triangle constructor, you should use `this.a`, `this.b`, and `this.c` to access the class properties. This will prevent reference errors since `a`, `b`, and `c` are not defined in the constructor scope.